custom_components/rivian/update.py

```diff
@@ -19,7 +19,7 @@
 from .coordinator import VehicleCoordinator
 from .entity import RivianVehicleEntity
 
-INSTALLING_STATUS = ("Install_Countdown", "Awaiting_Install", "Installing")
+INSTALLING_STATUS = ("Install_Countdown", "Awaiting_Install", "Installing", "Install_Failed")
```

Comment:
The "Install Failed" should be added to const.py for the `ota_status` sensor. I don't think we want to report it as still installing if it is install failed